app/views/posts/_write_post.haml

```diff
@@ -25,6 +25,8 @@
     - unless post.id.present? && !post.editable_by?(current_user)
       .view-button#rtf{class: ('selected' if current_user.default_editor == 'rtf')} Rich Text
       .view-button#html{class: ('selected' if current_user.default_editor == 'html')} HTML
+      .view-button#markup-allowed{title: "Toggle RTF editor and HTML editor\nAllowed tags: #{Glowfic::ALLOWED_TAGS.sort * ', '}"} ?
+      - # title gets overridden by JS
```

Comment:
Because it's extremely long and this is for NoScript users. When Javascript is enabled, it truncates the title but the 'spoiler' box is usable and contains the same / more information.